charts/bulk-scan-processor/values.aat.template.yaml

```diff
@@ -15,3 +15,24 @@ java:
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
     PROCESS_PAYMENTS_ENABLED: "true"
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    BULK_SCANNING_DB_USER_NAME: bulk_scanner@bulk-scan-processor-aat
```

Comment:
AAT has too many values, is it not possible to template them like https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L6 in values.yaml so that u dont' need to override in every environment.